apps/snitch_core/test/data/schema/permissson_test.exs

```diff
@@ -9,7 +9,7 @@ defmodule Snitch.Data.Schema.PermissionTest do
   }
 
   test "creation fails if code not provided" do
-    params = Map.drop(@params, [:code])
+    params = %{}
     %{valid?: validity} = Permission.create_changeset(%Permission{}, params)
     refute validity
```

Comment:
You **must** assert on the exact match for `errors_on(changeset)`. And don't write the assert like this:
```elixir
assert %{...} = errors_on(cs)
```
Use `==` instead. Most of our tests use the `=` match operator, and this is bad as it is not strict, and allows LHS to be not equal from RHS.